frontend/src/Guest/RegisterForm.tsx

```diff
@@ -21,16 +22,25 @@ export default function RegisterForm() {
     const [email, setEmail] = useState("");
     const [password, setPassword] = useState("");
     const [confirmPassword, setConfirmPassword] = useState("");
+    const [error, setError] = useState("");
 
     const handleFormSubmit = (event: ChangeEvent<HTMLFormElement>) => {
         event.preventDefault();
+        if (!isValidEmail(email)) {
+            setError("Email is invalid");
+        } else {
+            setError("");
+        }
         postForm();
         setFirstName("");
         setLastName("");
         setEmail("");
         setPassword("");
         setConfirmPassword("")
     }
+    const isValidEmail= (email:string) => {
+        return /.@./.test(email);
```

Comment:
Maybe not the best regex for email validation, but checking an email adress is not that easy. 
`abc@abc` would be a a valid email adress, since you check the adress in the backend i would say thats enough for now